src-ui/src/app/components/document-detail/document-detail.component.ts

```diff
@@ -48,8 +48,17 @@ export class DocumentDetailComponent implements OnInit {
     tags: new FormControl([])
   })
 
+  @ViewChild('nav') nav: NgbNav
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jonaswinkler/paperless-ng/pullRequest?prid=7592509)
```suggestion
  @ViewChild("nav") nav: NgbNav
```